src/components/ReportActionItem/ReportPreview.js

```diff
@@ -114,11 +133,17 @@ function ReportPreview(props) {
         }
     }
 
+    let previewMessage;
     const managerName = ReportUtils.isPolicyExpenseChat(props.chatReport) ? ReportUtils.getPolicyName(props.chatReport) : ReportUtils.getDisplayNameForParticipant(managerID, true);
+    if (isScanning) {
+        previewMessage = props.translate('common.receipt');
+    } else {
+        previewMessage = props.translate(iouSettled || props.iouReport.isWaitingOnBankAccount ? 'iou.payerPaid' : 'iou.payerOwes', {payer: managerName});
+    }
```

Comment:
Can we create a function to return `previewMessage`? I think that'd be cleaner and more scalable